pkg/kubectl/cmd/convert.go

```diff
@@ -189,12 +189,24 @@ func (o *ConvertOptions) RunConvert() error {
 		if err != nil {
 			return err
 		}
+
+		for _, item := range items {
+			actualVersion := item.GetObjectKind().GroupVersionKind()
```

Comment:
The short declared var only works inside the for() loop locally.
> actualVersion := item.GetObjectKind().GroupVersionKind()

https://golang.org/ref/spec#Short_variable_declarations
So this change can't compile because actualVersion in line 208 is not declared.